libs/common/src/assets/i18n/en.json

```diff
@@ -786,6 +786,7 @@
 	"stash.widgets.filter.help": "Displays filters in search bar",
 	"stash.widgets.filter.label": "Filter",
 	"stash.widgets.filter_type": "Filter types",
+	"stash.widgets.help": "It will take precedence on the user prompt defined in the Knowledge Box settings. Note: the prompt strategy might change depending on the generative model. Check the examples in the settings page.",
```

Comment:
I feel like you forgot a part in the key, shouldn't it be `stash.widgets.prompt.help`?